src/main/java/uk/gov/hmcts/reform/pip/account/management/service/AccountService.java

```diff
@@ -148,10 +148,10 @@ public Map<CreationEnum, List<? extends AzureAccount>> addAzureAccounts(
      * Method to add users to P&I database, loops through the list and validates the email provided then adds them to
      * success or failure lists.
      * @param users the list of users to be added.
-     * @param issuerEmail the email of the admin adding the users for logging purposes.
+     * @param issuerId the email of the admin adding the users for logging purposes.
```

Comment:
You have missed this one. The description should say "the id of the admin" not email :)